docs/docs/gatsby-config.md

```diff
@@ -32,13 +32,13 @@ module.exports = {
 }
 ```
 
-This way you can store it in one place, and pull it whenever you need it. If you ever need to update the info, you only have to change it here.
+Dzięki temu można je przechowywać w jednym miejscu, a następnie pobierać gdziekolwiek chcemy. Jeżeli kiedykolwiek pojawiła by się potrzeba zmiany tych danych, wystarczy zmienić je tylko w tym miejscu.
 
-See a full description and sample usage in [Gatsby.js Tutorial Part Four](/tutorial/part-four/#data-in-gatsby).
+Pełen opis i przykładowe użycie w [czwartej części poradnika Gatsby.js](/tutorial/part-four/#data-in-gatsby).
 
```

Comment:
Plugins means 'wtyczki'.

Reply:
Wszędzie przy tłumaczeniu wcześniejszych stron używałem słowa plugin z polską odmianą ;)

Reply:
W tabelce ustaliliśmy jednak jako Wtyczki :( widocznie jakoś przegapiliśmy w poprzednich PRach. Jak zmienisz to będzie git :)